app/actions/action_pins.js

```diff
@@ -1,5 +1,37 @@
-export default function dropNewPin() {
+import { DROP_NEW_PIN } from '../constants/constants.js';
+import {store } from 'redux';
+let nextPinId = 0;
+
+
+// const fire = new Firebase("https://interruptedlobster.firebaseio.com/user1");
+
+function dropNewPin(pinInfo) {
   return {
-    type: DROP_NEW_PIN
+    type: DROP_NEW_PIN,
+    id: nextPinId++,
+    long: pinInfo.long,
+    lat: pinInfo.lat,
```

Comment:
we should use 'longitude' and 'latitude' to be consistent with the data we're receiving from the navigation component